python/ray/_raylet.pyx

```diff
@@ -3465,11 +3474,40 @@ cdef class CoreWorker:
                             generator_id=CObjectID.Nil(),
                             owner_address=c_owner_address))
 
+    def put_serialized_object_to_mutable_plasma_object(self, serialized_object,
+                                                       ObjectRef object_ref,
+                                                       num_readers,
```

Comment:
Document this arg as experimental?

Reply:
I think this one's okay because this method is only called by the new experimental path.